### PR TITLE
End user/feature/statusbar

### PR DESCRIPTION
**Pull request by NadaFeteiha**
Simulator Screen Shot - iPhone 14 - 2023-09-09 at 14 49 24
Simulator Screen Shot - iPhone 14 - 2023-09-09 at 14 49 10
Simulator Screen Shot - iPhone 14 - 2023-09-09 at 14 48 51
Simulator Screen Shot - iPhone 14 - 2023-09-09 at 14 48 48
Screenshot 2023-09-09 at 2 48 16 PM
Screenshot 2023-09-09 at 2 48 02 PM
Screenshot 2023-09-09 at 2 47 55 PM

### CODE REVIEW

**Review by mosayed01 — APPROVED** [review]
Good job 👏

**Review by ay0o5h — APPROVED** [review]

File: client_end_user/androidApp/src/androidMain/kotlin/com/beepbeep/MainActivity.kt [hunk 1]
```diff
@@ -4,13 +4,18 @@ import MainView
...
 import androidx.appcompat.app.AppCompatActivity
+import androidx.core.view.WindowCompat
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
+        WindowCompat.setDecorFitsSystemWindows(window, false)
         super.onCreate(savedInstanceState)
 
         setContent {
+//            val systemUIController = rememberSystemUiController()
```
Comment (AzizaHelmy) [inline client_end_user/androidApp/src/androidMain/kotlin/com/beepbeep/MainActivity.kt:15]: if we don't need this comments, remove it

File: client_end_user/shared/build.gradle.kts [hunk 2]
```diff
@@ -72,6 +72,8 @@ kotlin {
                 api(libs.androidx.core.ktx)
                 api(libs.koin.android)
                 implementation(libs.androidx.constraint)
+                implementation(libs.google.accompanist)
```
Comment (AzizaHelmy) [inline client_end_user/shared/build.gradle.kts:75]: as i know accompanist libirary became deprecated or we don't need it to handle status bar, "edge to edge" is the replacement of it, check this:
https://developer.android.com/develop/ui/views/layout/edge-to-edge

**Review by msalarab — APPROVED** [review]
Well done 👏

File: client_end_user/shared/build.gradle.kts (hunk 2 above)
Comment (AzizaHelmy) [reply client_end_user/shared/build.gradle.kts:75]: https://developer.android.com/jetpack/compose/layouts/insets

**Review by ahmednasserzaza — APPROVED** [review]
Well done

File: client_end_user/shared/build.gradle.kts (hunk 2 above)
Comment (NadaFeteiha) [reply client_end_user/shared/build.gradle.kts:75]: @AzizaHelmy I'll try now to change it

**Pull request merged**